.github/workflows/pre-commit.yml

```diff
@@ -8,6 +8,8 @@ on:
 jobs:
   pre-commit:
     runs-on: ubuntu-latest
+    env:
+      SKIP: no-commit-to-branch
```

Comment:
this hook is just so that you don't accidentally make a commit to the `master` branch when developing locally, it needs to be skipped during CI as after PRs are merged the jobs are run from the `master` branch